tools/netqtop.py

```diff
@@ -28,7 +29,7 @@ def to_str(num):
     elif num > 1000:
         return str(round(num/1024.0, 2)) + 'K'
     else:
-        if type(num) == types.FloatType:
+        if isinstance(num, float):
```

Comment:
Since we removed using `types` module, we do not need to import it (line 9), right?

Reply:
yes, thank you. I deleted this line.